src/test/java/org/apache/commons/collections4/map/TransformedSortedMapTest.java

```diff
@@ -77,18 +79,15 @@ public void testTransformedMap() {
             map.put((K) els[i], (V) els[i]);
             assertEquals(i + 1, map.size());
             assertTrue(map.containsKey(Integer.valueOf((String) els[i])));
-            try {
-                map.containsKey(els[i]);
-                fail();
-            } catch (final ClassCastException ex) {}
+            SortedMap<K, V> finalMap1 = map;
+            int finalI = i;
```

Comment:
Why did you need these two variables above? :thinking:

Reply:
Otherwise you get this error "Variable used in lambda expression should be final or effectively final"